### PR TITLE
Python3 support + Cleaning up

### PR DESCRIPTION
**Pull request by CosmicWebServices**

### CODE REVIEW

**Comment by skistaddy** [comment]
Ok

**Comment by skistaddy** [comment]
It looks to me like you're trying to update it to Python 3 in the first file.
In the second, it starts like that, except you then stop half way through the file, and start using Python 2 again. I'm going to -1 on this. 👎

File: demo.py [hunk 1]
```diff
@@ -4,4 +4,4 @@
 pyfancy().multi("Multicolored text").output()
 pyfancy().rainbow("Rainbow text").output()
 pyfancy().parse("{red foo {bold bar} baz}").output()
-print pyfancy().red("Hello").raw(", ").blue("world!").strip()
+print(pyfancy().red("Hello").raw(", ").blue("world!").strip())
```
Comment (skistaddy) [inline demo.py:7]: What are you doing here? You ended the parentheses for `print` in the `strip` method.

File: pyfancy.py [hunk 2]
```diff
@@ -1,85 +1,107 @@
...
     # Returns output text and resets properties
+
     def get(self):
         return self.out + "\033[0m"
 
     # Outputs text using print (should work in Python 2 and 3)
+
     def output(self):
-        print(self.get())
+        print self.get()
```
Comment (skistaddy) [inline pyfancy.py]: What is this? In the first file you were doing Python 3.

Comment (CosmicWebServices) [reply pyfancy.py]: Oh boo I am sorry

File: demo.py (hunk 1 above)
Comment (CosmicWebServices) [reply demo.py:7]: Fixed

Comment (CosmicWebServices) [reply demo.py:7]: wait sorry... there is nothing wrong with this

**Comment by CosmicWebServices** [comment]
bump?

File: demo.py (hunk 1 above)
Comment (jeandrek) [reply demo.py:7]: @baranskistad I think you just misunderstood the edit because of the way GitHub highlighted it.

Comment (skistaddy) [reply demo.py:7]: Yeah, sorry bout that. :P

**Pull request closed without merging**